e2e/_suites/ingest-manager/features/agent_endpoint_integration.feature

```diff
@@ -0,0 +1,38 @@
+@agent_endpoint_integration
+Feature: Fleet Mode Agent with Endpoint Integration
+  Scenarios for Agent to deploy Endpoint and sending data to Ingest Manager and Elasticsearch.
+
+@deploy-endpoint-with-agent
+Scenario: Deploy Agent and add Endpoint Integration and host shows in Security App
+  Given an agent is deployed to Fleet
+  When the "Endpoint" latest package version is installed successfully
+    And the "Endpoint" integration is "added" in the "default" configuration
+  Then the "default" configuration shows the "Endpoint" datasource added
+    And the host name is shown in the security app
```

Comment:
Consistency
```suggestion
    And the host name is shown in the Security App
```

Reply:
We need the API calls to check this, exactly the same as we currently do for online agents in Fleet